quiz.js

```diff
@@ -26,6 +38,16 @@ if (square(2) !== 4 || cube(3) !== 27 || quad(4) !== 256) {
 
 /* your code starts here */
 
+function sum(numbers, twiceOrHalf) {
+  if (twiceOrHalf === true) {
+    return 2*numbers;
```

Comment:
you're on the right track,but you never actually sum up the numbers in the array. trying using a loop and counter next time. for example 
```
var counter = 0; 
  for (i=0; i < array.length; i++){ 
     count += array[i];
  }
```